src/ship/PlayerShipController.cpp

```diff
@@ -516,11 +502,14 @@ void PlayerShipController::StaticUpdate(const float timeStep)
 			ApplyTotalAction(act);
 			break;
 		case CONTROL_MANUAL:
+			// maintain the thrust set by the station when undocking while the timeout lasts
+			// but only in manual mode
+			if (m_ship->m_launchLockTimeout > 0) return;
```

Comment:
Just as a confirmation, have you tested that AI ships are still able to launch and fly a safe distance above the pad before rotating and hyperjumping?